CMakeLists.txt

```diff
@@ -194,21 +194,24 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "BSD")
 	set(BSD TRUE)
 	if(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
 		set(FREEBSD TRUE)
+		set(EPOLLSHIM TRUE)
 	endif()
 	if(${CMAKE_SYSTEM_NAME} MATCHES "kFreeBSD")
 		set(KFREEBSD TRUE)
 	endif()
 	if(${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD")
 		set(OPENBSD TRUE)
+		set(EPOLLSHIM TRUE)
 	endif()
 endif()
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "DragonFly")
 	set(BSD TRUE)
 	set(FREEBSD TRUE)
+	set(EPOLLSHIM TRUE)
 endif()
 
-if(FREEBSD)
+if(EPOLLSHIM)
```

Comment:
Where is ` EPOLLSHIM` defined? I can only find a `Find` module when searching for this, but not default in ` CMake`